src/utils.ts

```diff
@@ -457,16 +461,12 @@ export async function handleExternalProject(
       return;
```

Comment:
Since the return value is string, we can just return empty string ("") instead of undefined.

Reply:
The return value is Promise<void>, not string.